pfcpiface/bess.go

```diff
@@ -429,12 +429,12 @@ func (b *bess) sim(u *upf, method string) {
 
 		fars := []far{farDown, farN6Up, farN9Up}
 
-		switch timeout := 100 * time.Millisecond; method {
+		switch method {
 		case "create":
-			b.simcreateEntries(pdrs, fars, timeout)
+			b.sendMsgToUPF("add", pdrs, fars)
 
```

Comment:
does it also resolve Insert "time out issue"

Reply:
this will make the timeout consistent in all cases simulate vs real pfcp insert.